src/tcpip/driver_stm32.c

```diff
@@ -212,7 +211,7 @@ void ETH_IRQHandler(void) {
   ETH->DMARPDR = 0;     // and resume RX
 }
 
-struct mg_tcpip_driver mg_tcpip_driver_stm32 = {
-    mg_tcpip_driver_stm32_init, mg_tcpip_driver_stm32_tx, mg_tcpip_driver_rx,
-    mg_tcpip_driver_stm32_up};
```

Comment:
No changes needed if we keep the current API.
You did not take into account the RP2040 RMII example, then we decided to change the API as the new one was more homogeneous, flexible, and consistent.